### PR TITLE
Readme updated with description of server

### PR DESCRIPTION
**Pull request by Earthmark**
Updated the readme to describe common nouns and technical requirements.

### CODE REVIEW

File: LockPatron/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -35,6 +35,7 @@ public IQueryable<WeatherForecast> Get([FromQuery, Range(0, 50)] int count)
     [HttpPost]
     public async Task<WeatherForecast> Post()
     {
+      _logger.LogTrace("Adding new forecast.");
```
Comment (TehTurk) [inline LockPatron/Controllers/WeatherForecastController.cs]: Cloudy with a chance of?

Comment (Kal47) [inline LockPatron/Controllers/WeatherForecastController.cs]: It looks like some other code got added in the commit.

Comment (Earthmark) [reply LockPatron/Controllers/WeatherForecastController.cs]: Yepper dee doo! Wrong place for logs.

Comment (Earthmark) [reply LockPatron/Controllers/WeatherForecastController.cs]: Yep, was an old commit, now it's pruned.

**Review by TehTurk — APPROVED** [review]

**Review by Kal47 — APPROVED** [review]
ENpEtWNWwAM6Z0-
Nice

**Comment by TehTurk** [comment]
https://www.google.com/url?sa=i&url=https%3A%2F%2Fknowyourmeme.com%2Fmemes%2Flaughing-lizard-hhhehehe&psig=AOvVaw2TlBJDuDS7SbUBYqtGaMWk&ust=1595556239411000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCIiisKik4uoCFQAAAAAdAAAAABAO

**Pull request merged**